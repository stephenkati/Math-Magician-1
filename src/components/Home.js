const Home = () => (
  <>
    <div className="homeContent">
      <p>Hello and Welcome to this calculator webapp.</p>
      <p>
        In the calculator page you will find the calculator application, have fun with some math
      </p>
      <p>In the quote page you will find some motivational, insightful quotes. Enjoy</p>
    </div>
  </>
);

export default Home;
