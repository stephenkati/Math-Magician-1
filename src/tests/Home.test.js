import renderer from 'react-test-renderer';
import Home from '../components/Home';

describe('test for home', () => {
  test('test if home matches', () => {
    const homeComponent = renderer
      .create(<Home />)
      .toJSON();
    expect(homeComponent).toMatchSnapshot();
  });
});
