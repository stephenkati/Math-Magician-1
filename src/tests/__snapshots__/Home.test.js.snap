// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for home test if home matches 1`] = `
<div
  className="homeContent"
>
  <p>
    Hello and Welcome to this calculator webapp.
  </p>
  <p>
    In the calculator page you will find the calculator application, have fun with some math
  </p>
  <p>
    In the quote page you will find some motivational, insightful quotes. Enjoy
  </p>
</div>
`;
